chef/automate
# Moving the License Banner

### :nut_and_bolt: Description: What code changed, and why?
- Added logic to display notifications on different parts of the page from each other. 
- Move the license expiration banner to the top of the page to align with original designs and prepare for an additional banner on the bottom of the page for projects. 
*Note: This is for the expiring license banner only, all other user notifications should stay where they are.*

### :+1: Definition of Done
- [x] - Move the banner to the very top of the page, all other content should shift down
- [x] - Change the content to say: There are days remaining on your license. Apply a license or contact Chef to continue to use Chef Automate.
- [x] - The x-button should dismiss the banner and shift the rest of the content back up

### :athletic_shoe: How to Build and Test the Change
1.      build components/automate-ui-devproxy/
2.      start_automate_ui_background
3.      start_all_services 
4.      chef-automate dev psql chef_license_control_service -- -c "UPDATE licenses SET active=NULL"
5.  Add a user and see that other types of notifications work and remain at the original spot

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
